src/include/txn_inline.h

```diff
@@ -2072,7 +2073,9 @@ __txn_modify_block(
     F_CLR(txn, WT_TXN_IGNORE_PREPARE);
     for (; upd != NULL && !__wt_txn_upd_visible(session, upd); upd = upd->next) {
         if (upd->txnid != WT_TXN_ABORTED) {
-            __wt_verbose_debug1(session, WT_VERB_TRANSACTION,
+            ++txn->modify_block_count;
+            __wt_verbose_level(session, WT_VERB_TRANSACTION,
+              txn->modify_block_count >= 100 ? WT_VERBOSE_INFO : WT_VERBOSE_DEBUG_1,
```

Comment:
Can we make the value 100 configurable? It would be helpful to define it as a constant, similar to WT_THOUSAND, so that we can easily change it in one place if needed, rather than modifying every instance where it is used.